### PR TITLE
Fixed issue with large amount of ids in query

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | none
| Related issues/PRs | none
| License | MIT
| Documentation PR | none

#### What's in this PR?

This PR fixes a problem when using csv rest-route to export large-number of entities.

#### Why?

The SuluCommentBundle throws error when there are 70000 entities.

#### To Do

- [ ] Tests

### CODE REVIEW

**Comment by alexander-schranz** [comment]
There is a different between the result of the query when example using GROUP CONCAT and a filter on it:

New:

```sql
SELECT f.id, f.name, t.name, GROUP_CONCAT(t.name) FROM me_file_versions f
LEFT JOIN me_file_version_tags ft ON 
    f.id = ft.idFileVersions
LEFT JOIN ta_tags t ON
    t.id = ft.idTags
WHERE f.id IN (576, 1050) AND t.name = 'Weltreise'
GROUP BY f.id
```

bildschirmfoto 2018-02-26 um 17 10 55


Old:

```sql
/* Without ID subquery */
SELECT f.id, f.name, t.name, GROUP_CONCAT(t.name) FROM me_file_versions f
LEFT JOIN me_file_version_tags ft ON 
    f.id = ft.idFileVersions
LEFT JOIN ta_tags t ON
    t.id = ft.idTags
WHERE f.id IN (

/* Id Query */
SELECT f2.id FROM me_file_versions f2
LEFT JOIN me_file_version_tags ft2 ON 
    f2.id = ft2.idFileVersions
LEFT JOIN ta_tags t2 ON
    t2.id = ft2.idTags
    WHERE t2.name = 'Weltreise'
 ) 

GROUP BY f.id
```

bildschirmfoto 2018-02-26 um 17 11 16

**Comment by wachterjohannes** [comment]
@danrot what do you think?

File: src/Sulu/Component/Rest/ListBuilder/Doctrine/DoctrineListBuilder.php [hunk 1]
```diff
@@ -156,27 +160,39 @@ public function execute()
         }
 
         // now select all data
-        $this->queryBuilder = $this->em->createQueryBuilder()
-            ->from($this->entityName, $this->entityName);
+        $this->queryBuilder = $this->createFullQueryBuilder(
+            $this->em->createQueryBuilder()->from($this->entityName, $this->entityName)
+        );
+
+        // now select all data
```
Comment (danrot) [inline src/Sulu/Component/Rest/ListBuilder/Doctrine/DoctrineListBuilder.php]: Same comment twice

**Comment by danrot** [comment]
Looks good else. Can you also rebase?

**Pull request merged**